docs/helmi.md

```diff
@@ -39,7 +39,13 @@ The coupler characteristics are given below. TC refers to tunable coupler.
 | --------------------------------- | ------ | ------ | ------ | ------ |
 | Two-qubit Gate Fidelities[^3] [%] | 96.4   | 95.9   | 94.6   | 97.5   |
 
-The PRX gate length and the CZ gate length is 120 [ns] respectively.
+The CZ gate length are:
```

Comment:
A note should be added that the gate lengths may vary and how can a user query them